README.md

```diff
@@ -1,25 +1,24 @@
-
 # XTB_trading_bot
 
 XTB_trading_bot is exactly what its name implies. A bot written in Python, works with XTB broker using the API.py file (a wrapper for XTB API, not written by me).
 
-
 ## Documentation
 
 At this moment, the bot doesn't trade by itself, it only takes the tickers available from the XTB broker, applies a RSI + BB strategy and sends all tickers for Buy and Sell through a channel in Telegram.
 The original link for the wrapper is here https://github.com/caiomborges/Python-XTB-API but I decided to included it in my files for a quicker setup.
 I'm not gonna lie, my journey in Python is in its infancy steps, so at the moment ChatGPT is a great help to unblock me and improve my code where needed.
 
-
 ## Setup
 
 After installing all the libraries needed (sorry, at this point I don't have enough experience with pip or such, but I will work on making a requirements.txt), you will need to setup a telegram bot channel in order for you to receive the notifications send by the bot.
  - In the API.py wrapper you will need to modify the file at line 553 for which kind of XTB account you are using, demo or real.
```

Comment:
That line might change constantly, not sure if is that good to add a static line nr. here.

Reply:
in time, i will make it dynamic, but for now, it's just for quick guidance